### PR TITLE
feat(sql): alter table .. add/drop offline_path

### PR DESCRIPTION
**Pull request by aceforeverd**
## Syntax

```yacc
"ALTER" "TABLE" <table_path> [ "ADD" | "DROP" ] OFFLINE_PATH string_literal
```

### CODE REVIEW

File: hybridse/include/vm/physical_op.h [hunk 1]
```diff
@@ -1828,6 +1829,31 @@ class PhysicalCreateTableNode : public PhysicalOpNode {
...
+    base::Status InitSchema(PhysicalPlanContext *) override { return base::Status::OK(); }
+    base::Status WithNewChildren(node::NodeManager *nm, const std::vector<PhysicalOpNode *> &children,
+                                 PhysicalOpNode **out) override {
+        return base::Status::OK();
+    }
+
+    static PhysicalAlterTableNode *CastFrom(PhysicalOpNode *node);
+
+    const std::string& db() const { return db_; }
+    const std::string& table() const { return table_; }
```
Comment (aceforeverd) [inline hybridse/include/vm/physical_op.h]: expose `actions_`

File: cases/plan/cmd.yaml [hunk 2]
```diff
@@ -584,3 +584,33 @@ cases:
         +-node[CMD]
           +-cmd_type: exit
           +-args: []
+
+  - id: alter-table-add-path
+    sql: |
+      alter table t1 add path '12'
```
Comment (aceforeverd) [inline cases/plan/cmd.yaml]: `path` -> `offline_path`

File: hybridse/include/vm/physical_op.h [hunk 3]
```diff
@@ -1828,6 +1829,32 @@ class PhysicalCreateTableNode : public PhysicalOpNode {
     static PhysicalCreateTableNode *CastFrom(PhysicalOpNode *node);
 };
 
+class PhysicalAlterTableNode : public PhysicalOpNode {
```
Comment (aceforeverd) [inline hybridse/include/vm/physical_op.h]: physical won't used

File: cases/plan/cmd.yaml [hunk 4]
```diff
@@ -584,3 +584,33 @@ cases:
         +-node[CMD]
           +-cmd_type: exit
           +-args: []
+
+  - id: alter-table-add-path
```
Comment (aceforeverd) [inline cases/plan/cmd.yaml:588]: add case for multiple alter actions

**Review by dl239 — APPROVED** [review]

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**